README.md

```diff
@@ -30,21 +31,74 @@ const { response } = await hcaptcha.execute({ async: true });
+| `assethost`       | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `endpoint`        | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `hl`              | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `host`            | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `imghost`         | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `recaptchacompat` | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `reportapi`       | String      | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `sentry`          | Boolean     | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+| `custom`          | Boolean     | No       | `-`                                | See enterprise docs.                                                                                                                                      |
+
+
+
+## Legacy Support
+In order to support older browsers, a separate bundle is generated in which all ES6 code is compiled down to ES5 along with an optional polyfill bundle.
+
+- `polyfills.js`: Provides polyfills for features not supported in older browsers.
+- `index.es5.js`: **@hcaptcha/loader** package compiled for ES5 environments.
+
+### Import Bundle(s)
+Both bundles generated use IIFE format rather than a more modern import syntax such as `require` or `esm`.
+
+```js
+// Optional polyfill import
+import '@hCaptcha/loader/dist/polyfills.js';
+// ES5 version of hCaptcha Loader
+import '@hCaptcha/loader/dist/index.es5.js';
+
+window['hCaptchaLoader']().then(function(hcaptcha) {
```

Comment:
Couldn't we keep this as
**hCaptchaLoader.then** without the window[]?

Reply:
This can be used as
- `hCaptchaLoader()`
- `window.hCaptchaLoader()`
- `window['hCaptchaLoader']`

Third option works with TS out of the box, i'll update the readme